src/main/resources/config.yml

```diff
@@ -93,58 +93,89 @@ allow:
 #   - 'n:b:/ban:_'
 #
 blocked_commands:
-    # Disabled commands
+  # Disabled commands
   - 'n:b:/time:Server-side time changing is disabled. Please use /ptime to set your own personal time.'
-  - 'n:b:/gamemode:Use /creative and /survival to set your gamemode.'
+  - 'n:b:/gamemode:Use /gmc and /gms to set your gamemode.'
   - 'n:b:/ban:_'
   - 'n:b:/pardon:_'
   - 'n:b:/ban-ip:_'
+  - 'n:b:/viaversion debug:_'
   - 'n:b:/pardon-ip:_'
   - 'n:b:/toggledownfall:_'
   - 'n:b:/effect:Please use /potion to set effects.'
-  - 'n:b:/enderchest:_'
   - 'n:b:/spreadplayers:_'
   - 'n:b:/execute:_'
+  - 'n:b:/emf:_'
   - 'n:b:/blockdata:_'
   - 'n:b:/clearhistory:_'
   - 'n:b:/save-on:_'
   - 'n:b:/save-off:_'
-  - 'n:b:/playsound:_'
+  - 'n:b:/skin set:_'
+  - 'n:b:/calculate:_'
+  - 'n:b:/clone:_'
   - 'n:b:/skinrestorer:_'
+  - 'n:b:/testfor:_'
+  - 'n:b:/testforblock:_'
+  - 'n:b:/testforblocks:_'
   - 'n:b:/packet:_'
+  - 'n:b://eval:_'
   - 'n:b:/thaw:_'
-  - 'n:b:/pp largeexplode:You may not use this particle! Please use another.'
-  - 'n:b:/pp hugeexplosion:You may not use this particle! Please use another.'
-  - 'n:b:/pp barrier:You may not use this particle! Please use another.'
+  - 'n:b:/spawnmob:_'
   - 'n:b:/setjail:_'
-  - 'n:b:/tfbanlist:Use /banlist to see and wipe the ban list'
-  - 'n:b:/tfipbanlist:Use /banlist to see and wipe the ban list'
 
   # Superadmin commands
   - 's:b:/save-all:_'
+  - 's:b:/summon:Do to security reasons, the use of /summon has been disabled.'
```

Comment:
This is simply the current config on the server. I will advise Mark that the corrections are needed, once he makes the change I will update this PR with the change.